git_os_job/cmd.py

```diff
@@ -20,6 +20,50 @@
 import sys
 import webbrowser
 
+import six
+
+
+def safe_decode(text, incoming=None, errors='strict'):
```

Comment:
This seems overly complicated. We know we're getting bytes back from subprocess, right? Or at least we know the conditions under which we would get something other than bytes.